docs: Update docs about busola-kyma image (#3402)
kyma-project/busola: README.md

For the information on how to run tests and configure them, go to the [`tests`](tests) directory.

## Busola in Docker

> ### Adding a cluster via kubeconfig ID

Reviewer: ```suggestion
### Adding a cluster using kubeconfig ID
```

2. When you open Busola in your browser, go to `http://localhost:3001?kubeconfigID={YOUR_KUBECONFIG_FILE_NAME}`. Busola will try to download that file and add it for your Busola instance.

### Set active environment

> 1. To use one of build-in environments in `busola-kyma` image (dev, stage, prod), pass env `ENVIRONMENT` to docker container.

Reviewer: ```suggestion
1. To use one of the built-in environments in the `busola-kyma` image (dev, stage, prod), pass env `ENVIRONMENT` to the Docker container.
```

   ```bash
   docker run --rm -it -p 3001:3001 -v --env ENVIRONMENT={your-env}--pid=host --name busola europe-docker.pkg.dev/kyma-project/prod/busola-kyma-web:latest
   ```

> 2. To use custom environment configuration, mount it in docker and pass env `ENVIRONMENT` to docker container.

Reviewer: ```suggestion
2. To use a custom environment configuration, mount it in Docker and pass the `ENVIRONMENT`  env to the Docker container.
```
